lib/qesapdeployment.pm

```diff
@@ -349,19 +350,20 @@ sub qesap_execute {
 
     my $verbose = $args{verbose} ? "--verbose" : "";
     my %paths = qesap_get_file_paths();
-    my $qesap_cmd = join(" ", $paths{deployment_dir} . "/scripts/qesap/qesap.py", $verbose, "-c", $paths{qesap_conf_trgt}, "-b", $paths{deployment_dir});
     my $exec_log = "/tmp/qesap_exec_" . $args{cmd} . ".log.txt";
-    push(@log_files, $exec_log);
-
-    record_info('QESAP exec', 'Executing: \n' . $qesap_cmd . " " . $args{cmd});
-
-    if ($args{timeout}) {
-        assert_script_run(join(" ", $qesap_cmd, $args{cmd}, "2>&1 | tee -a", $exec_log), timeout => $args{timeout});
-    }
-    else {
-        assert_script_run(join(" ", $qesap_cmd, $args{cmd}, "2>&1 | tee -a", $exec_log));
-    }
+    my $qesap_cmd = join(" ", $paths{deployment_dir} . "/scripts/qesap/qesap.py",
+        $verbose,
+        "-c", $paths{qesap_conf_trgt},
+        "-b", $paths{deployment_dir},
+        $args{cmd},
+        $args{cmd_options},
+        "2>&1 | tee -a",
```

Comment:
Just tip, `|&` can be used instead `2>&1 |`

Reply:
Cool it works:
https://mordor.suse.cz/tests/4935/#
Test failed but command was executed correctly:
```
# /root/qe_sap_deployment/scripts/qesap/qesap.py --verbose -c /root/qe_sap_deployment/scripts/qesap/hanasr_azure.yaml -b /root/qe_sap_deployment ansible  |& tee -a /tmp/qesap_exec_ansible.log.txt; echo RMEEv-$?-
```